src/shared/dissect-image.c

```diff
@@ -275,6 +276,20 @@ static int device_monitor_handler(sd_device_monitor *monitor, sd_device *device,
         if (device_for_action(device, SD_DEVICE_REMOVE))
                 return 0;
 
+        if (w->uevent_seqnum_not_before != UINT64_MAX) {
+                uint64_t seqnum;
+
+                r = sd_device_get_seqnum(device, &seqnum);
+                if (r < 0)
+                        goto finish;
+
+                if (seqnum <= w->uevent_seqnum_not_before) { /* From an older use of this loop device */
```

Comment:
Hmm, so the naming is "off by one" here: "not before" would be appropriate if the check was
`seqnum < w->uevent_seqnum_not_before`.
For consistency, the field should be `uevent_seqnum_after`.